shared/chat/selectable-big-team-channel.js

```diff
@@ -69,6 +69,8 @@ class FilterBigTeamChannel extends PureComponent<Props, State> {
   }
 }
 
+export const rowHeight = isMobile ? 64 : 56
```

Comment:
why invert this relationship?

Reply:
You mean why have the inbox one import this but not the other way around? Because this is where the component is defined, and the inbox filtered row uses this component, so I feel like this is the correct place for the size as well.

Reply:
hmm. well the original idea was all the row sizes would be in there so its clear looking at the components that their sizes actually aren't defined in themselves, so you know there's some other entity thats tracking this. else you'll think its just some regular component and its not that clear that it exports these sizes for some reason

Reply:
I had to move this up because it'll be used by `chat/conversation-list` as well. What do you think if we move `chat/inbox/row/sizes.js` to `chat/sizes.js`, and still track all size in there?